internal/config/yaml_parser_test.go

```diff
@@ -81,6 +84,9 @@ func (t *YamlParserTest) TestReadConfigFile_ValidConfig() {
 	AssertEq(ERROR, mountConfig.LogConfig.Severity)
 	AssertEq("/tmp/logfile.json", mountConfig.LogConfig.FilePath)
 	AssertEq("text", mountConfig.LogConfig.Format)
+	AssertEq(100, mountConfig.LogConfig.LogRotate.MaxSizeInMB)
```

Comment:
Is there a lower/upper limit on `mountConfig.LogConfig.LogRotate.MaxSizeInMB` or on `mountConfig.LogConfig.LogRotate.BackupCount` or on combinations of those two ? If yes, we should add tests for those.